test/common_test.py

```diff
@@ -1,5 +1,4 @@
-"""General tests to be performed on most or all classes."""
-from test.conftest import SET_FIXTURE_MAP
+"""Sets of assertions for testing the saving and retrieval of sets."""
```

Comment:
This is now just functions to check that the output of save and get queries is as expected -- no actual tests.